Dockerfile.dev

```
@@ -0,0 +1,22 @@
+# Use the specified image
+FROM node:14-buster-slim
```

Node.js 14 has already reached the end-of-life, and [Node.js 16 is going to reach EOL shortly](https://github.com/nodejs/Release#release-schedule).

I suggest using `node:16-buster-slim` since it looks from the contents of `package.json`
```
"@types/node": "^16.9.1",
```

```suggestion
FROM node:16-buster-slim
```

We can address transition from 16 to 18 (the next LTS version) in another PR.